docs/Create-Packages/Creating-a-Package.md

```diff
@@ -236,22 +236,21 @@ When using a convention-based working directory, on the other hand, NuGet assume
+- Transformations to source code files in a project using `.pp` files.
 
 The folder conventions are as follows:
 
 | Folder | Description | Action upon package install |
 | --- | --- | --- |
 | (root) | Location for readme.txt | Visual Studio displays a readme.txt file in the package root when the package is installed. |
-| lib | Assembly (`.dll`), documentation (`.xml`), and symbol (`.pdb`) files | Assemblies are added as references; `.xml` and `.pdb` copied into project folders. See [Supporting multiple target frameworks](Supporting-Multiple-Target-Frameworks.md) for creating framework target-specific sub-folders. |
-| runtimes | Architecture-specific assembly (`.dll`), symbol (`.pdb`), and native resource (`.pri`) files | Assemblies are added as references; other files are copied into project folders. See [Supporting multiple target frameworks](Supporting-Multiple-Target-Frameworks.md). |
-| content | Arbitrary files | Contents are copied to the project root |
-| build | MSBuild `.targets` and `.props` files | Automatically inserted into the project file (NuGet 2.x) or `project.lock.json` (NuGet 3.x+). |
-| tools | Powershell scripts and programs accessible from the Package Manager Console | Contents are copied to the project folder, and the `tools` folder is added to the PATH environment variable. |
-
-Think of the **content** folder as the root of the target application. To have the package add an image in the application's */images* folder, place it in the package's *content/images* folder.
+| lib | Assembly (`.dll`), documentation (`.xml`), and symbol (`.pdb`) files | Assemblies are added as references; additional files in this folder are extracted to disk in the global packages folder. See [Supporting multiple target frameworks](Supporting-Multiple-Target-Frameworks.md) for creating framework target-specific sub-folders. |
+| runtimes | Architecture-specific assembly (`.dll`), symbol (`.pdb`), and native resource (`.pri`) files | Assemblies are added as references; other files are extracted in the `packages` folder. See [Supporting multiple target frameworks](Supporting-Multiple-Target-Frameworks.md). |
+| contentFiles | Arbitrary files | Contents are extracted in the `packages` folder and linked to target project based on buildAction. See [Including content files](../schema/nuspec.md#including-content-files). |
+| build | MSBuild `.targets` and `.props` files | Automatically inserted into the project file (when using `packages.config`), `project.lock.json` (when using `project.json`), or `<projectName>.csproj.nuget.g.targets`/ `<projectName>.csproj.nuget.g.props` (when using PackageReference and referenced for specific TFM). |
+| buildMultiTargeting | MSBuild `.targets` and `.props` files | Only applicable for PackageReference; this is a build folder that does not use a TFM, it is referenced in the outer build only for multi-framework projects. |
```

Comment:
This is correct, but I would add `(not commonly used, use build unless you know that you need this folder)` I think users might get confused and use this all the time instead of build, it would seem to just work, but it would have problems in some scenarios.